src/egon/data/datasets/sanity_checks.py

```diff
@@ -0,0 +1,460 @@
+    AND country = 'DE')
+    AND c.scenario_name='eGon2035'
+    GROUP BY (scn_name, a.carrier, c.capacity);
+    
+    """
+    )
+
+    sum_installed_storage_cap_DE["Error"] = (
+        (
+            sum_installed_storage_cap_DE["capacity_mw"]
+            - sum_installed_storage_cap_DE["target_capacity"]
+        )
+        / sum_installed_storage_cap_DE["target_capacity"]
+    ) * 100
+
+    g1 = sum_installed_storage_cap_DE["Error"].values[0]
+    g = round(g1, 2)
+
+    print(f"The target values for Pumped Hydro differ by {g}  %")
+
+    print("Results for summary of loads in DE grid:")
+
+    sum_loads_DE_grid = db.select_dataframe(
+        f"""
+    SELECT a.scn_name, a.carrier,  ROUND((SUM((SELECT SUM(p) FROM UNNEST(b.p_set) p))/1000000)::numeric, 2) as load_twh
+    FROM grid.egon_etrago_load a
+    JOIN grid.egon_etrago_load_timeseries b
+    ON (a.load_id = b.load_id)
+    JOIN grid.egon_etrago_bus c
+    ON (a.bus=c.bus_id)
+    AND b.scn_name = 'eGon2035'
+    AND a.scn_name = 'eGon2035'
+    AND c.scn_name= 'eGon2035'
+    AND c.country='DE'
+    GROUP BY (a.scn_name, a.carrier);
+    
+    
+    """
+    )
+
+    sum_loads_DE_AC_grid = sum_loads_DE_grid["load_twh"].values[1]
+    sum_loads_DE_heat_grid = (
+        sum_loads_DE_grid["load_twh"].values[2]
+        + sum_loads_DE_grid["load_twh"].values[4]
+    )
+
+    sum_loads_DE_demand_regio_cts_ind = db.select_dataframe(
+        f"""
+    SELECT scenario, ROUND(SUM(demand::numeric/1000000), 2) as demand_mw_regio_cts_ind
+    FROM demand.egon_demandregio_cts_ind
+    WHERE scenario= 'eGon2035'
+    AND year IN ('2035')
+    GROUP BY (scenario);
+    
+    
+    """
+    )
+    demand_regio_cts_ind = sum_loads_DE_demand_regio_cts_ind[
+        "demand_mw_regio_cts_ind"
+    ].values[0]
+
+    sum_loads_DE_demand_regio_hh = db.select_dataframe(
+        f"""
+    SELECT scenario, ROUND(SUM(demand::numeric/1000000), 2) as demand_mw_regio_hh
+    FROM demand.egon_demandregio_hh
+    WHERE scenario= 'eGon2035'
+    AND year IN ('2035')
+    GROUP BY (scenario);
+    
+    
+    """
+    )
+
+    demand_regio_hh = sum_loads_DE_demand_regio_hh[
+        "demand_mw_regio_hh"
+    ].values[0]
+
+    sum_loads_DE_AC_demand = demand_regio_cts_ind + demand_regio_hh
+
+    sum_loads_DE_demand_peta_heat = db.select_dataframe(
+        f"""
+    SELECT scenario, ROUND(SUM(demand::numeric/1000000), 2) as demand_mw_peta_heat
+    FROM demand.egon_peta_heat
+    WHERE scenario= 'eGon2035'
+    GROUP BY (scenario);
+    
+    
+    """
+    )
+
+    sum_peta_heat_DE_demand = sum_loads_DE_demand_peta_heat[
+        "demand_mw_peta_heat"
+    ].values[0]
+
+    Error_AC = (
+        round(
+            (sum_loads_DE_AC_grid - sum_loads_DE_AC_demand)
+            / sum_loads_DE_AC_demand,
+            2,
+        )
+        * 100
+    )
+
+    print(f"The target values for Sum loads AC DE differ by {Error_AC}  %")
+
+    demand_peta_heat = sum_loads_DE_demand_peta_heat[
```

Comment:
If I remember right, we preferred to have separate functions for electricity and heat.